astro/src/content/docs/lifecycle/authenticate-users/_bootstrapping-sso.mdx

```diff
@@ -0,0 +1,21 @@
+---
+---
+import Aside from 'src/components/Aside.astro';
+import InlineField from 'src/components/InlineField.astro';
+
+<Aside type="version">Available in FusionAuth 1.56.0</Aside>
+
+In certain scenarios you may have authenticated a user outside of the typical OAuth flow. In such cases you can still acquire an SSO session from FusionAuth as long as you have a valid access token for the user. To do this, take the user's browser to the `/oauth2/authorize` endpoint and provide the access token as a `Bearer` token in the `Authorization` header. If the token is valid, FusionAuth will create an SSO session and redirect the browser to the `redirect_uri` you provided. The `redirect_uri` must be an authorized URI, registered with the given application. See the [OAuth Authorize](/docs/lifecycle/authenticate-users/oauth/endpoints#authorize) documentation for details.
```

Comment:
`In such cases` -> "If so,"
`take the user's browser to` -> "navigate to"

I think there is a flow issue here. Telling folks to navigate to an endpoint and provide an access token, and then not mentioning that they'll need to add the header using a reverse proxy (or other) until 2 paragraphs later may cause confusion. I recommend moving the section about browsers and authorization headers up into this paragraph.